ext/node/polyfills/_fs/_fs_writeFile.ts

```diff
@@ -42,7 +42,7 @@ export function writeFile(
     optOrCallback instanceof Function ? undefined : optOrCallback;
 
   if (!callbackFn) {
-    throw new TypeError("Callback must be a function.");
+    throw new TypeError("Callback must be a function");
```

Comment:
This change is nice, but one thing about `ext/node` is probably the error messages should align with whatever node does for better compability? By doing this are we making the compatiblity worse or are these error messages already not aligned with node?

Reply:
That's a good question. It seems they are not the same either way:

```
fs.writeFile('/Users/irbull/test.txt', content, undefined, undefined);
```

In node (v21.6.2) I get:
```
TypeError [ERR_INVALID_ARG_TYPE]: The "cb" argument must be of type function. Received undefined
```

In Deno (1.46.3) I get:
```
TypeError: Callback must be a function.
```

Reply:
I'm happy to pump the brakes on the node compat stuff. I don't know if we should consider the error messages part of the API surface. It seems like it might have to be, but then we should align with what Node does. That would be a pretty big undertaking. Have you seen reports of inconsistencies causing problems?